plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -569,7 +569,7 @@ class read_write {
 
    using push_block_params = chain::signed_block;
    using push_block_results = empty;
-   void push_block(const push_block_params& params, chain::plugin_interface::next_function<push_block_results> next);
+   void push_block(push_block_params params, chain::plugin_interface::next_function<push_block_results> next);
```

Comment:
This could be `void push_block(push_block_params&& params, chain::plugin_interface::next_function<push_block_results> next);`